samples/msal-node-samples/ElectronSystemBrowserTestApp/src/Main.ts

```diff
@@ -58,141 +68,138 @@ export default class Main {
         Main.createMainWindow();
-        Main.mainWindow.loadFile(path.join(__dirname, "../index.html"));
+        Main.mainWindow.loadURL(MAIN_WINDOW_WEBPACK_ENTRY);
         Main.mainWindow.on("closed", Main.onClose);
         Main.authProvider = new AuthProvider(Main.authConfig);
         Main.fetchManager = new FetchManager();
         Main.registerSubscriptions();
-
         Main.attemptSSOSilent();
     }
 
+    private static onClose(): void {
+        Main.mainWindow = null;
+    }
+
     // Creates main application window
     private static createMainWindow(): void {
         this.mainWindow = new BrowserWindow({
             width: APPLICATION_DIMENSIONS.WIDTH,
             height: APPLICATION_DIMENSIONS.HEIGHT,
+
             /**
              * Preload script serves as an interface between the Main process
              * that has access to Node API and the Renderer process which controls
              * the user interface but is otherwise not trustworthy of directly handling
              * the Node API.
              */
-            webPreferences: { preload: path.join(__dirname, "preload.js") },
+            webPreferences: {
+                preload: MAIN_WINDOW_PRELOAD_WEBPACK_ENTRY,
+                nodeIntegration: false,
+            },
         });
     }
 
     private static publish(message: string, payload: any): void {
         Main.mainWindow.webContents.send(message, payload);
     }
 
+    
     private static async attemptSSOSilent(): Promise<void> {
-        const account = await Main.authProvider.loginSilent();
+        const tokenRequest = {
+            scopes: authConfig.resourceApi.scopes,
+            account: null as AccountInfo,
+        };
+        const account = await Main.authProvider.loginSilent(tokenRequest);
         await Main.loadBaseUI();
 
         if (account) {
             console.log("Successful silent account retrieval");
+            Main.mainWindow.minimize();
+            Main.mainWindow.focus();
             Main.publish(IpcMessages.SHOW_WELCOME_MESSAGE, account);
         }
     }
 
-    private static async getCodeFromDeepLinkUrl(url: string): Promise<void> {
-        const account = await Main.authProvider.getTokenByCode(url);
-        if (account) {
-            await Main.loadBaseUI();
-            Main.publish(IpcMessages.SHOW_WELCOME_MESSAGE, account);
-        }
+    private static async loadBaseUI(): Promise<void> {
+        await Main.mainWindow.loadURL(MAIN_WINDOW_WEBPACK_ENTRY);
     }
 
     private static async login(): Promise<void> {
-        await Main.authProvider.login();
+        try {
+            const account = await Main.authProvider.login();
+            if (account) {
+                await Main.loadBaseUI();
+                Main.mainWindow.minimize();
+                Main.mainWindow.focus();
+                Main.publish(IpcMessages.SHOW_WELCOME_MESSAGE, account);
+            }
+        } catch (error) {
+            console.log(error, " error");
```

Comment:
use template literal, but also this isn't consistent with the other catches in this sample where you don't print the string at the end

Reply:
Done!